src/ui/rcb-ui.svelte

```diff
@@ -91,6 +96,10 @@
     new Translator(customTranslations, selectedLocale, defaultLocale),
   );
 
+  setContext(eventsTrackerContextKey, eventsTracker);
```

Comment:
Maybe is better to have a `TrackingContext` interface that contains all of those and set the context only once.

Something like
```
interface TrackingContext{
   tracker: EventsTracker;
   appUserId: string;
   userIsAnonymous: boolean;
}

[...]

setContext(trackingContextKey, {
   tracker: eventsTracker,
   appUserId: appUserId,
   userIsAnonymous: useIsAnonymous
})
```

In this way we have to propagate less keys down the nested components and we do less calls to getContext